src/lib/moment/compare.js

```diff
@@ -29,7 +29,14 @@ export function isBefore (input, units) {
     }
 }
 
-export function isBetween (from, to, units) {
+export function isBetween (from, to, units, inclusivity) {
+    if (inclusivity === '(]') {
```

Comment:
I love using ternary operators. The tradeoff is readability, but I don't think that is a problem here. Other than the mistype at the end, it looks solid. `inclusivity = inclusivity || '()'` takes care of inclusivity ever being `undefined` and it has a smaller footprint. I wrote the commit to be representative of the current code base; but like I said, I have a soft spot for ternary operators.